.github/workflows/tests.yaml

```
@@ -9,66 +9,75 @@ on:
       - 3.5.x
 
 jobs:
-  Tests:
+  tests:
+    name: Tests
     runs-on: ubuntu-latest
-    continue-on-error: ${{ matrix.experimental == true }}
     env:
-      SYMFONY_REQUIRE: ${{matrix.symfony_constraint}}
+      SYMFONY_REQUIRE: ${{ matrix.symfony-version }}
     strategy:
       fail-fast: false
       matrix:
         php:
-          - '8.0'
-          - '7.4'
-          - '7.3'
           - '7.2'
-        sentry_constraint: [false]
-        symfony_constraint: ['']
-        experimental: [false]
-        include:
-#          - description: 'sentry/sentry dev-develop'
-#            php: '7.4'
-#            sentry_constraint: 'dev-develop'
-#            experimental: true
-          - description: 'Symfony 4.4'
-            php: '7.3'
-            symfony_constraint: 4.4.*
-          - description: 'Symfony 3.4'
-            php: '7.2'
-            symfony_constraint: 3.4.*
-            env:
-              SYMFONY_DEPRECATIONS_HELPER: disabled
-          - description: 'prefer lowest'
-            php: '7.2'
-            composer_option: '--prefer-lowest'
-            env:
-              SYMFONY_DEPRECATIONS_HELPER: disabled
-
-    name: PHP ${{ matrix.php }} tests (${{ matrix.description }})
+          - '7.3'
+          - '7.4'
+          - '8.0'
+        symfony-version:
+          - 3.4.*
+          - 4.4.*
+          - 5.2.*
```

I would prefer it like this:
```suggestion
          - 5.*
```
So we don't have to bump it every six months due to minor releases.

---

Right, I didn't think about it! Change applied